install.sh

```diff
@@ -17,6 +22,16 @@ debian_distribution_name=""
 sfx_ingest_url="https://ingest.signalfx.com"
 dimensions=""
 input_collectd=""
+# Number of seconds system clock can be out of sync from NTP server.
+time_sync_max_drift=10
+
+is_bsd() {
```

Comment:
you could also use this method in other places that is doing same thing.  see determine_os